app/assets/javascripts/components/grade_submission_form.js.jsx

```diff
@@ -16,7 +16,6 @@ var Nav = React.createClass({
     render: function () {
         return (
             <div>
-                <Link url='/instructor/dashboard' text="Return to Dashboard"/>
```

Comment:
It's broken. I tried to fix it by passing it in as a variable but my react skills are bad.